tests/spread/core24/linters/library/task.yaml

```diff
@@ -1,14 +1,13 @@
 summary: Test library linter output
 
 restore: |
-  # If this next line fails, delete it and replace it with the one following it
-  snapcraft clean 2>&1 | MATCH '"clean" command is not implemented for core24!'; exit 0
+  snapcraft clean 2>&1 | MATCH 'snapcraft internal error: KeyError'; exit 0
   # snapcraft clean
   rm -f ./*.snap ./*.txt
 
 execute: |
   # If this next line fails, delete it and replace it with the one following it
-  snapcraft 2>&1 | MATCH '"pack" command is not implemented for core24!'; exit 0
+  snapcraft 2>&1 | MATCH 'snapcraft internal error: KeyError'; exit 0
```

Comment:
This seems like an odd error to be expecting. Can you explain why it's doing that?

Reply:
It's a craft-application bug that only showed up here because [the project doesn't have a version](https://github.com/canonical/craft-application/blob/main/craft_application/application.py#L567). I'll create a separate craft-app bug for it, but it's going to (or should be) addressed when we implement adopt-info/dynamic versioning